components/raftstore-v2/src/operation/command/mod.rs

```diff
@@ -183,16 +184,26 @@ impl<EK: KvEngine, ER: RaftEngine> Peer<EK, ER> {
 
     #[inline]
     fn propose<T>(&mut self, ctx: &mut StoreContext<EK, ER, T>, data: Vec<u8>) -> Result<u64> {
-        ctx.raft_metrics.propose.normal.inc();
+        self.propose_with_proposal_ctx(ctx, data, vec![])
```

Comment:
the name propose_with_proposal_ctx can be renamed as propose_with_ctx

Reply:
done